### PR TITLE
NOTIF-129 Adds bundle's facet and udpates the Facet type to contain

### PR DESCRIPTION
**Pull request by josejulio**
displayName, name and id

### CODE REVIEW

**Comment by josejulio** [comment]
@gwenneg could you review?

File: src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java [hunk 1]
```diff
@@ -378,4 +379,36 @@ void testGetEventTypesAffectedByEndpoint() {
...
+        String localIdentityHeaderValue = TestHelpers.encodeIdentityInfo(tenant, userName);
+        Header localIdentityHeader = TestHelpers.createIdentityHeader(localIdentityHeaderValue);
+        mockServerConfig.addMockRbacAccess(localIdentityHeaderValue, RbacAccess.READ_ACCESS);
+        given()
+                .header(localIdentityHeader)
+                .when()
+                .contentType(ContentType.JSON)
+                .get("/notifications/facets/applications?bundleName=insights")
+                .then()
+                .statusCode(200);
```
Comment (gwenneg) [inline src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java:395]: Does the response contain some facets that could be checked here or is the response empty?

File: src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java [hunk 2]
```diff
@@ -378,4 +379,36 @@ void testGetEventTypesAffectedByEndpoint() {
...
+        String localIdentityHeaderValue = TestHelpers.encodeIdentityInfo(tenant, userName);
+        Header localIdentityHeader = TestHelpers.createIdentityHeader(localIdentityHeaderValue);
+        mockServerConfig.addMockRbacAccess(localIdentityHeaderValue, RbacAccess.READ_ACCESS);
+        given()
+                .header(localIdentityHeader)
+                .when()
+                .contentType(ContentType.JSON)
+                .get("/notifications/facets/bundles")
+                .then()
+                .statusCode(200);
```
Comment (gwenneg) [inline src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java:411]: Same question here.

File: src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java (hunk 1 above)
Comment (josejulio) [reply src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java:395]: It should have "policies" at least.

File: src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java (hunk 2 above)
Comment (josejulio) [reply src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java:411]: It should have `insights` at least.

**Comment by gwenneg** [comment]
Great, thanks!

LGTM

**Review by gwenneg — APPROVED** [review]

**Pull request merged**